src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -84,6 +84,36 @@ Common questions and requests include:
 
 Our tech writers in Barcelona cover the 2nd shift heroing during their regular working hours. Unlike the US-based heroes, our Barcelona writers hero for an entire two-week sprint. Second-shift heroes cover both GitHub and Slack, but we don't expect that second-shift heroes will field every single request that comes in during their working hours since they're also carrying standard sprint duties during their hero shift.
 
+## Route and triage user feedback
+
+The hero is responsible for routing userfeedback to liasions. This sections covers the entire lifecycle of a user feedback ticket:
+
+1. The hero is responsible for routing incoming tickets to the proper liaison each day
+  - The Hero leaves a comment saying the ticket is being triaged
+2. Each writer should spend 10-15 minutes/day triaging their assigned tickets 
```

Comment:
This is assuming we'll have work, maybe more like 'up to 15 minutes or so'. Right? Cause they might not have anything, right?